src/server/routes/orderRoutes.js

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+import { getAllOrders, createOrder } from '../controllers/orderController.js';
+
+const router = express.Router();
```

Comment:
change name for each routr that you do to him export. 
cant be the same name for many vars